### PR TITLE
 Replace metadata-code-themes.scss & Build with nodejs/npm saas and react-scripts

### PR DESCRIPTION
**Pull request by rxm7706**
This is the best I could do...and is a draft for review.. Will update the notes here - if the builds pass. 
Locally I was able to test successfully against https://github.com/conda-forge/kedro-viz-feedstock/issues/3
@millsks @NeroOkwa @astrojuanlu - this might help debug / fix some of the upstream issues. 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,29 +11,52 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install ./package -vv
-  number: 0
+  script:
```
Comment (astrojuanlu) [inline recipe/meta.yaml:14]: As far as I understand, a `.sh` build script can be provided separately, see for example

https://github.com/conda-forge/fenics-feedstock/blob/12ef077802cc9fad34cf984ec7af80585b44301b/recipe/meta.yaml#L180-L186

Comment (astrojuanlu) [reply recipe/meta.yaml:14]: So, these instructions could be a `build.sh`

**Review by astrojuanlu — COMMENTED** [review]
Thanks @rxm7706! How can I test this?

File: recipe/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipe/meta.yaml:14]: Yes - for sure !

**Comment by millsks** [comment]
I will dop some local testing on my side.  If there are no problems I would suggest merging it so that there is a working version on conda-forge.

**Comment by rxm7706** [comment]
I did some local testing - and had no issues ; ok with merging to have a working version on conda-forge for 6.3.4.

**Comment by millsks** [comment]
Ran the build and tested locally.  These changes worked.

``` bash
(kedro310-conda) millsks@ATREIDES:~/UserLocal/repos/kedro-viz-feedstock-rxm7706$ mamba install --use-local kedro-viz

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.4.1) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['kedro-viz']

warning  libmamba Cache file "/home/millsks/UserLocal/mambaforge/pkgs/cache/497deca9.json" was modified by another program
warning  libmamba Cache file "/home/millsks/UserLocal/mambaforge/pkgs/cache/09cdf8bf.json" was modified by another program
warning  libmamba Could not parse state file: Could not load cache state: [json.exception.type_error.302] type must be string, but is null        
warning  libmamba Could not parse state file: Could not load cache state: [json.exception.type_error.302] type must be string, but is null        
home/millsks/UserLocal/mambaforge/conda-bld/linu..  12.3kB @ 228.0MB/s  0.0s
home/millsks/UserLocal/mambaforge/conda-bld/noarch   2.1kB @  62.3MB/s  0.0s
selfexplainml/linux-64                               5.2kB @   9.8kB/s  0.5s
selfexplainml/noarch                                32.3kB @  21.4kB/s  1.5s
conda-forge/noarch                                  13.7MB @   2.0MB/s  7.0s
conda-forge/linux-64                                33.6MB @   2.7MB/s 12.5s

Pinned packages:
  - python 3.10.*


Transaction

  Prefix: /home/millsks/UserLocal/mambaforge/envs/kedro310-conda

  Updating specs:

   - kedro-viz
   - ca-certificates
   - certifi
   - openssl


  Package             Version  Build            Channel                                                   Size
────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  Install:
────────────────────────────────────────────────────────────────────────────────────────────────────────────────

  + httptools           0.6.0  py310h2372a71_0  conda-forge/linux-64                                    Cached
  + kedro-viz           6.3.4  py_1             /home/millsks/UserLocal/mambaforge/conda-bld/noarch     Cached
  + libuv              1.44.2  hd590300_1       conda-forge/linux-64                                    Cached
  + python-dotenv       1.0.0  pyhd8ed1ab_0     conda-forge/noarch                                      Cached
  + uvicorn-standard   0.22.0  hff52083_0       conda-forge/linux-64                                    Cached
  + uvloop             0.17.0  py310h5764c6d_1  conda-forge/linux-64                                    Cached
  + watchfiles         0.19.0  py310h4426083_0  conda-forge/linux-64                                    Cached
  + websockets         11.0.3  py310h2372a71_0  conda-forge/linux-64                                    Cached

  Summary:

  Install: 8 packages

  Total download: 0 B

────────────────────────────────────────────────────────────────────────────────────────────────────────────────


Confirm changes: [Y/n]
```


``` bash
$ mamba list|grep kedro-viz
kedro-viz                 6.3.4                      py_1    local
```


``` bash
(kedro310-conda) millsks@ATREIDES:~/UserLocal/projects/spaceflights$ kedro info

 _            _
| | _____  __| |_ __ ___
| |/ / _ \/ _` | '__/ _ \
|   <  __/ (_| | | | (_) |
|_|\_\___|\__,_|_|  \___/
v0.18.12

Kedro is a Python framework for
creating reproducible, maintainable
and modular data science code.

Installed plugins:
kedro_telemetry: 0.2.5 (entry points:cli_hooks,hooks)
kedro_viz: 6.3.4 (entry points:global,line_magic)
```


``` python
$ python
Python 3.10.12 | packaged by conda-forge | (main, Jun 23 2023, 22:40:32) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import kedro_viz
>>> kedro_viz.__version__
'6.3.4'
>>> kedro_viz.__file__
'/home/millsks/UserLocal/mambaforge/envs/kedro310-conda/lib/python3.10/site-packages/kedro_viz/__init__.py'
>>> exit()
```

``` python
(kedro310-conda) millsks@ATREIDES:~/UserLocal/projects/spaceflights$ kedro viz
WARNING: You are using an old version of Kedro Viz. You are using version 6.3.4; however, version 6.4.0 is now available.
You should consider upgrading via the `pip install -U kedro-viz` command.
You can view the complete changelog at https://github.com/kedro-org/kedro-viz/releases.
[08/18/23 13:09:12] WARNING  /home/millsks/UserLocal/mambaforge/envs/kedro310-conda/lib/python3.10/site-packages/kedro/framework/s warnings.py:109
                             ession/session.py:266: FutureWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the   
                             OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here:              
                             https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader          
                               warnings.warn(   
```


image

**Comment by rxm7706** [comment]
LGTM - @astrojuanlu - are you going to test too... or can this be merged.. 
best case - this works, worst case - it remains not working... 
Do you have any additional feedback / concerns.

**Comment by astrojuanlu** [comment]
I can't test this right now, go ahead and merge it! I'll give the built version a try

**Pull request merged**